styles.css

```diff
@@ -14,15 +15,29 @@ h1 {
   padding-right: 20%;
 }
 
+h2 {
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  margin: 8px 0 0;
+}
+
 a {
   text-decoration: none;
 }
 
+hr {
+  border-color: #dfe1e6;
+  opacity: 0.4;
+}
+
 .pf-wrap {
   max-width: 375px;
-  min-height: 718px;
+  height: auto;
   margin: 5vh auto 5vh auto;
-  background-image: url("../portfolio-website/img/header-background-mobile.png");
+  background-image: url("./img/header-background-mobile.png");
+  background-repeat: no-repeat;
+  background-color: #e5e5e5;
```

Comment:
- [ ] I notice a grayish color at the top and bottom of your project, which is not supposed to be like that. I would suggest you change the CSS property in line 37 from `margin: 5vh auto 5vh auto;` to `margin: 0 auto;`. With this you will achieve removing the extra color showing at the top and bottom of your project

Your project 👇 

![far1](https://user-images.githubusercontent.com/78634841/212253706-e98c3feb-68e1-42d6-97b1-bffbfe8c5eba.png)